WpfTest/View/LoginView.xaml.cs

```diff
@@ -12,19 +13,19 @@ namespace WpfTest.View
     /// </summary>
     public partial class LoginView : Window
     {
-        private ApplicationContext db;
-        private readonly LoginValidator validator; 
+        ///private ApplicationContext db;
```

Comment:
please do not send comments to review, you have to delete them, you have history of your code so you can revert code if needed

Reply:
Thanks for the advice - it was really helpful!